Source/Core/loadAndExecuteScript.js

```diff
@@ -0,0 +1,32 @@
+define([
+    '../ThirdParty/when'
+], function(
+    when) {
+        'use strict';
+
+    /**
+     * @private
+     */
+    function loadAndExecuteScript(url) {
+        var deferred = when.defer();
+        var script = document.createElement('script');
+        script.async = true;
+        script.src = url;
+
+        var head = document.getElementsByTagName('head')[0];
```

Comment:
Isn't there a (small) chance the page may not have a `head` element? Can we just append it to the document?

Reply:
The `head` element will always exist.